docs/tutorials/anomaly-detection.rst

```diff
@@ -0,0 +1,542 @@
+Streaming anomaly detection with Apache Flink®, Apache Kafka® and PostgreSQL®
+==============================================================================================
+
+.. Note::
+
+    This tutorial doesn't assume any existing Apache Kafka®, PostgreSQL® or Apache Flink® knowledge
+
+What you will learn
+---------------------------
+
+The tutorial provides an example of a streaming anomaly detection system, by following it you'll learn:
+
+* How to create a fake streaming dataset
+* How to create and use Apache Kafka® for data streaming
+* How to create and use PostgreSQL® to store threshold data
+* How to create and use Apache Flink® to define streaming data pipelines
+* How to push the outcome of the anomaly detection system as a Slack notification
+
+
+What are you going to build
+---------------------------
+
+In this tutorial you will build a streaming anomaly detection system based on IoT sensor readings recording the CPU usage percentage for various devices. After setting up the various components and enabling the data flow, you'll then create a basic filtering pipeline to separate the usage values surpassing a fixed threshold (80%). 
+
+.. mermaid::
+
+    graph LR;
+
+        id1(IoT device)-- sensor reading -->id2(usage > 80%?);
+        id2-- yes -->id3(notification);
+
+However, receiving a notification on every sensor reading that surpasses a fixed threshold can be overwhelming and create false positives for short usage spikes. Therefore you'll create a second, more advanced, pipeline to average the sensor readings values over 30 seconds windows and then compare the results with host based thresholds coming from an external table.
+
+.. mermaid::
+
+    graph LR;
+
+        id1(IoT device)-- sensor reading -->id2(average 30 seconds);
+        id2-- average reading -->id4(over threshold?);
+        id4-- yes --> id5(notification);
+        id3(threshold table)-- thresholds --> id4;
+
+.. Note::
+
+    Even if the sample dataset and the data pipeline examples might seem basic, they offer a wide coverage on the integration and transformation options available with Apache Flink that can be applied to other, more complex, scenarios. 
+
+The tutorial includes:
+
+* Apache Flink for data transformation
+* Apache Kafka for data streaming
+* PostgreSQL® for data storage/query
+* Slack as notification system
+
+.. Tip::
+
+    All the tools listed above are fully open source, and can therefore be installed and used locally. You're going to use the comparable Aiven services to avoid any network and integration complexities. 
+
+Architecture overview
+---------------------
+
+The tutorial showcases how to create an Apache Kafka® source topic that provides a stream of IoT metrics data, a PostgreSQL® database that contains data on the alerting thresholds, and an Apache Flink® service that combines these two services and pushes the filtered data to a separate Apache Kafka® topic or a Slack channel.
```

Comment:
This is similarly in a very dry language. Maybe it is the goal, but maybe we want to make it friendlier, like blog style? I suspect tutorials are more like blogs, rather than docs.